setup.py

```diff
@@ -19,7 +19,16 @@
 URL = ''
 LICENSE = 'BSD (3-clause)'
 DOWNLOAD_URL = 'http://github.com/jonescompneurolab/hnn-core'
-VERSION = '0.1.dev0'
+
+# get the version
+version = None
+with open(os.path.join('mne_bids', '__init__.py'), 'r') as fid:
```

Comment:
```suggestion
with open(os.path.join('hnn_core', '__init__.py'), 'r') as fid:
```
Haha :) ![images](https://user-images.githubusercontent.com/5610076/105666206-0b928d00-5ed9-11eb-84e0-267397cd509f.jpeg)